.github/workflows/deploy_mkdocs.yml

```diff
@@ -41,7 +37,7 @@ jobs:
             --output_dir docs/api/ \
             --exclude_source \
             --overwrite \
-            stac_fastapi
+            stac_fastapi.sqlalchemy
```

Comment:
@geospatial-jeff I wasn't sure whether to change this or not. It might be nice for developers to have all the docs they need in the same place. Also, I'm guessing that different backends are going to end up on different version of `stac-fastapi` at some point. If that's the case, it might not be enough to just point back to the core docs since those will always represent the most recent version.

Reply:
https://github.com/stac-utils/stac-fastapi-sqlalchemy/pull/1/commits/08c3764a522298c119858e0d579b715f33368e43